app/src/main/java/org/wikipedia/commons/FilePageView.kt

```diff
@@ -79,7 +79,7 @@ class FilePageView constructor(context: Context, attrs: AttributeSet? = null) :
     }
 
     private fun getImageTags(imageTags: Map<String, List<String>>, languageCode: String) : String? {
-        return imageTags[languageCode]?.joinToString(separator = "\n")
+        return imageTags[languageCode]?.joinToString(separator = ", ")
```

Comment:
Do all languages use an ASCII comma to separate items in a list? What about Chinese? Or Hebrew?

Reply:
We can use ASCII comma for Chinese, but I think it does not work for Hebrew. I'll let Robin know about this.